components/Login/OrgLogin.tsx

```diff
@@ -0,0 +1,99 @@
+import { LightBottomIcon, DarkBottomIcon } from '@/assets/Icons/auth/Icons';
+import { Text } from '@/components/Themed';
+import { OrgLoginSchema } from '@/validations/login.schema';
+import { Ionicons } from '@expo/vector-icons';
+import { router } from 'expo-router';
+import { useFormik } from 'formik';
+import { useState } from 'react';
+import { ActivityIndicator, TextInput, TouchableOpacity, useColorScheme, View } from 'react-native';
+import { SvgXml } from 'react-native-svg';
+
+type FormValues = {
+  organization: string;
+};
+
+type OrgLoginProps = {
+  onSubmit: (values: FormValues) => void;
+};
+
+export default function OrgLogin({ onSubmit }: OrgLoginProps) {
+  const [loading, setLoading] = useState(false);
+  const colorScheme = useColorScheme();
+  const textColor = colorScheme === 'dark' ? 'text-gray-100' : 'text-secondary-light-900';
+
+  const formik = useFormik<FormValues>({
+    initialValues: {} as FormValues,
+    onSubmit: async (values) => {
+      setLoading(true);
+      onSubmit(values);
```

Comment:
await for this function because it returns a promise

Reply:
Done.